### DISCUSSION TITLE
Besteht die Möglichkeit das RRDtool 1.2.30 auf RRDtool 1.8.0 gebracht wird

### DISCUSSION THREAD

**Discussion by freetz-ng-mod** [Freetz-NG/freetz-ng#539] [Unerfüllte Wünsche]
Wie der Titel schon sagt, würde ich gerne wissen, ob RRDtool ein Update auf RRDtool 1.8.0  bekommt.

**Comment by fda77** [comment]
Grundsätzlich ja, es muss nur jemand machen.
Ich hatte das auch vor nicht all zu langem versucht, es haben sich einige Libs geändert die benötigt werden. Ich hatte alles hinbekommen bis auf rrdgraph (der Teil der die Bilder erzeugt). Dieser benötigt cairo&pango, mit denen war irgend was komisch.
Diese sollten zuerst in Freetz integriert werden. Siehe auch https://oss.oetiker.ch/rrdtool/doc/rrdbuild.en.html
